internal/osbuildexecutor/runner-impl-aws-ec2.go

```diff
@@ -24,6 +30,174 @@ func prepareSources(manifest []byte, store string, extraEnv []string, result boo
 	return err
 }
 
+func waitForSI(ctx context.Context, host string) bool {
+	client := http.Client{
+		Timeout: time.Second * 1,
+	}
+
+	for {
+		resp, err := client.Get(fmt.Sprintf("%s/api/v1/", host))
+		if err != nil {
+			logrus.Debugf("Waiting for secure instance continues: %v", err)
```

Comment:
(super nitpick) we probably want to get into the habbit of writing `%w` for errors so that they can be unwrapped potentially (here and in similar places but that is probably it's own followup so that we are consistent :)